<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2560</width>
    <height>1400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressource/qlamz.qrc">
    <normaloff>:/icons/icons/29119-amazon.png</normaloff>:/icons/icons/29119-amazon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="xml">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableViewTracks">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::AllEditTriggers</set>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>25</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>25</number>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="CustomWebView" name="webViewWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QProgressBar" name="progressBarWebView">
          <property name="value">
           <number>100</number>
          </property>
          <property name="textVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTextEdit" name="labelLink">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(239, 239, 239);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCancel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>C&amp;ancel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDownload">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Download</string>
          </property>
          <property name="icon">
           <iconset resource="../ressource/qlamz.qrc">
            <normaloff>:/icons/icons/actions/download.png</normaloff>:/icons/icons/actions/download.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonQuit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>&amp;Quit</string>
          </property>
          <property name="icon">
           <iconset resource="../ressource/qlamz.qrc">
            <normaloff>:/icons/icons/actions/application-exit.png</normaloff>:/icons/icons/actions/application-exit.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Application">
    <property name="title">
     <string>&amp;Application</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentFiles">
     <property name="title">
      <string>Recent Files</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentFiles"/>
   </widget>
   <widget class="QMenu" name="menu_Extras">
    <property name="title">
     <string>&amp;Extras</string>
    </property>
    <widget class="QMenu" name="menuAmazonCookie">
     <property name="title">
      <string>Amazon Cookie</string>
     </property>
     <addaction name="actionCookieAmazonDe"/>
    </widget>
    <addaction name="menuAmazonCookie"/>
    <addaction name="actionAmazonStore"/>
    <addaction name="separator"/>
    <addaction name="actionErrorLog"/>
    <addaction name="actionShowXMLContent"/>
    <addaction name="separator"/>
    <addaction name="actionExportCookies"/>
   </widget>
   <addaction name="menu_Application"/>
   <addaction name="menu_File"/>
   <addaction name="menu_Extras"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSelectAll"/>
   <addaction name="actionDeselectAll"/>
   <addaction name="separator"/>
   <addaction name="actionAmazonStore"/>
  </widget>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/configure.png</normaloff>:/icons/icons/actions/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/application-exit.png</normaloff>:/icons/icons/actions/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/help-about.png</normaloff>:/icons/icons/actions/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About qlamz</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+?</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/document-open.png</normaloff>:/icons/icons/actions/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/dialog-ok-apply.png</normaloff>:/icons/icons/actions/dialog-ok-apply.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/actions/edit-delete.png</normaloff>:/icons/icons/actions/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Deselect All</string>
   </property>
  </action>
  <action name="actionCookieAmazonDe">
   <property name="text">
    <string>Amazon.de</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAmazonStore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressource/qlamz.qrc">
     <normaloff>:/icons/icons/29119-amazon.png</normaloff>:/icons/icons/29119-amazon.png</iconset>
   </property>
   <property name="text">
    <string>Amazon Store</string>
   </property>
   <property name="toolTip">
    <string>Open the amazon store webpage in your default Browser.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionErrorLog">
   <property name="text">
    <string>Show Errors</string>
   </property>
   <property name="toolTip">
    <string>Show the last error log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionShowXMLContent">
   <property name="text">
    <string>Show XML</string>
   </property>
  </action>
  <action name="actionExportCookies">
   <property name="text">
    <string>&amp;Export Cookies</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomWebView</class>
   <extends>QWidget</extends>
   <header>CustomWebView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressource/qlamz.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonQuit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDownload</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>cancelDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAmazonFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSettings</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDeselectAll</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deselectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelectAll</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCookieAmazonDe</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>cookieAmazonDe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutQt</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aboutQt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionErrorLog</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showErrorLog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAmazonStore</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAmazonStore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowXMLContent</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showXMLContent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportCookies</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportCookies()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openAmazonFile()</slot>
  <slot>openDownloadDir()</slot>
  <slot>updateDownloadButton()</slot>
  <slot>startDownload()</slot>
  <slot>cancelDownload()</slot>
  <slot>settings()</slot>
  <slot>selectAll()</slot>
  <slot>deselectAll()</slot>
  <slot>about()</slot>
  <slot>cookieAmazonDe()</slot>
  <slot>aboutQt()</slot>
  <slot>showErrorLog()</slot>
  <slot>openAmazonStore()</slot>
  <slot>showXMLContent()</slot>
  <slot>exportCookies()</slot>
 </slots>
</ui>
